pkg/kp/rollstore/store.go

```diff
@@ -176,3 +186,21 @@ func RollLockPath(rcId rcf.ID) (string, error) {
 
 	return path.Join(kp.LOCK_TREE, subRollPath), nil
 }
+
+func (s consulStore) kvpToRU(kvp *api.KVPair) (rollf.Update, error) {
+	ru := rollf.Update{}
+	err := json.Unmarshal(kvp.Value, &ru)
+	return ru, util.Errorf("Unable to unmarshal value as rolling update: %s", err)
+}
+
+func (s consulStore) kvpsToRUs(kvps []*api.KVPair) ([]rollf.Update, error) {
```

Comment:
i was copying what's done in rcstore, but you're right it doesn't make sense. removed